### DISCUSSION TITLE
Relation field display

### DISCUSSION THREAD

**Discussion by SenH** [pocketbase/pocketbase#51] [Q&A]
When adding a relation type field to a collection, it would be nice to choose a specific relation field (ex. project name or e-mail) to display in the table instead of the unique id.

**Comment by ganigeorgiev** [comment]
It's a reasonable request, but it may complicate managing the collection (both in the code and for the end user) in case the assigned displayable field is renamed or removed.

I may consider it in the future, but for now I prefer to keep things as simple as possible.
